app/assets/stylesheets/article-show.scss

```diff
@@ -395,20 +375,11 @@ article {
   .body {
     margin: auto;
     width: 82%;
-    font-family:
-      Palatino,
-      'Palatino Linotype',
-      'Palatino LT STD',
-      'Book Antiqua',
-      Georgia,
-      serif;
+    font-family: Palatino, 'Palatino Linotype', 'Palatino LT STD',
+      'Book Antiqua', Georgia, serif;
```

Comment:
Why is this split into two lines?

Reply:
It’s just prettier formatting. There is a max line width